fh-messaging/lib/mongo_functions/mbaas_reduce_apprequestsdest.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable*/
+//ts and emit are used for testing purposes
+var ts;
+var emit;
+function mbaas_reduce_apprequestsdest(k, mbaasMetrics) {
```

Comment:
A note on the map and reduce functions. These are passed to mongo and are invoked within the mongo environment so they need to be self contained functions.
There is no native `forEach`, `reduce` functions available.